config/marconi.mk

```diff
@@ -5,7 +5,7 @@ INCLUDE += -I$(NETCDF_INC) -I$(HDF5_INC)
 GLFLAGS  = -lm
 CC=icc
 MPICC=mpiicc
-OPT=-O3 -xHost
+OPT=-O3 -xHost -fma -xMIC-AVX512 -finline-functions -align
```

Comment:
OK then just move the -xMIC-AVX512 to feltor/config/devices/devices.mk with 
OPT+=-xMIC ...
and maybe remove the -mmic in the devices.mk file. I don't know if this is still relevant